server/service/src/processors/transfer/invoice/create_inbound_invoice.rs

```diff
@@ -31,7 +31,7 @@ impl InvoiceTransferProcessor for CreateInboundInvoiceProcessor {
 
     /// Inbound invoice will be created when all below conditions are met:
     ///
-    /// 1. Source invoice name_id is for a store that is active on current site (transfer processor driver guarantees this)
+    /// 1. Source invoice name_link_id is for a store that is active on current site (transfer processor driver guarantees this)
```

Comment:
Just using "name id" would be better (since name link id is suppose to be repo layer...)